Assets/Tests/PlayMode/ItemTest.cs

```diff
@@ -0,0 +1,311 @@
+using System.Collections;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+using UnityEngine.SceneManagement;
+using UnityEngine.UI;
+
+namespace Tests
+{
+    public class ItemTest
+    {
```

Comment:
Great PlayMode test. This is a good start to getting code coverage increased on Item and the derived classes.